frontend/src/pages/modelRegistry/screens/RegisterModel/utils.ts

```diff
@@ -154,6 +162,14 @@ export const isRegisterModelSubmitDisabled = (
 export const isRegisterVersionSubmitDisabled = (formData: RegisterVersionFormData): boolean =>
   !formData.registeredModelId || isSubmitDisabledForCommonFields(formData);
 
+export const isRegisterCatalogModelSubmitDisabled = (
+  formData: RegisterCatalogModelFormData,
+): boolean =>
+  !formData.modelRegistry ||
+  !formData.modelName ||
+  isSubmitDisabledForCommonFields(formData) ||
+  !isNameValid(formData.modelName);
```

Comment:
Since this form state extends the register model form state, I think this can just be
```ts
isRegisterModelSubmitDisabled(formData, []) || !formData.modelRegistry
```
and then we should add a comment explaining that the `[]` there is temporary and linking to the future issue for fixing the `isModelNameExisting` validation on the catalog registration page. Or if you prefer, have a `registeredModels` param here and pass in `[]` when calling `isRegisterCatalogModelSubmitDisabled`, and put the comment there.